executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/parser/sections/ProjectViewSingletonSectionParserTest.java

```diff
@@ -22,191 +23,205 @@
 // TODO to kotlin
 public class ProjectViewSingletonSectionParserTest<V, T extends ProjectViewSingletonSection<V>> {
 
-  public static Stream<Arguments> data() {
+public static Stream<Arguments> data() {
     return Stream.of(
-        bazelPathSectionArguments(), debuggerAddressSectionArguments(), javaPathSectionArguments());
-  }
+            bazelPathSectionArguments(), debuggerAddressSectionArguments(), javaPathSectionArguments());
+}
 
-  private static Arguments bazelPathSectionArguments() {
+private static Arguments bazelPathSectionArguments() {
     var parser = ProjectViewBazelPathSectionParser.INSTANCE;
     var rawValueConstructor = (Function<String, String>) (seed) -> "/path/to/bazel/" + seed;
     var sectionMapper =
-        (Function<Path, ProjectViewBazelPathSection>) ProjectViewBazelPathSection::new;
+            (Function<Path, ProjectViewBazelPathSection>) ProjectViewBazelPathSection::new;
     var elementMapper = (Function<String, Path>) Paths::get;
 
     var sectionConstructor =
-        createSectionConstructor(rawValueConstructor, sectionMapper, elementMapper);
+            createSectionConstructor(rawValueConstructor, sectionMapper, elementMapper);
     var sectionName = parser.getSectionName();
 
     return Arguments.of(parser, rawValueConstructor, sectionConstructor, sectionName);
-  }
+}
 
-  private static Arguments debuggerAddressSectionArguments() {
+private static Arguments debuggerAddressSectionArguments() {
     var parser = ProjectViewDebuggerAddressSectionParser.INSTANCE;
     var rawValueConstructor = (Function<String, String>) (seed) -> "host_" + seed + ":8080";
     var sectionMapper =
-        (Function<String, ProjectViewDebuggerAddressSection>)
-            ProjectViewDebuggerAddressSection::new;
+            (Function<String, ProjectViewDebuggerAddressSection>)
+                    ProjectViewDebuggerAddressSection::new;
     var elementMapper = (Function<String, String>) x -> x;
 
     var sectionConstructor =
-        createSectionConstructor(rawValueConstructor, sectionMapper, elementMapper);
+            createSectionConstructor(rawValueConstructor, sectionMapper, elementMapper);
+    var sectionName = parser.getSectionName();
+
+    return Arguments.of(parser, rawValueConstructor, sectionConstructor, sectionName);
+}
+
+private static Arguments buildManualTargetsSectionArguments() {
+    var parser = ProjectViewBuildManualTargetsSectionParser.INSTANCE;
+    var rawValueConstructor = (Function<String, String>) (seed) -> "false" + seed;
```

Comment:
well it would create very strange things like `falsevalue` so lets return false or true always (I'll be updating this test soon anyway)